pubsub/lib/pubsub_samples.ex

```diff
@@ -4,6 +4,31 @@ defmodule GoogleApi.PubSub.Samples do
   """
 
   @doc """
+  Get a PubSub topic.
+
+  ## Examples
+
+      iex> GoogleApi.PubSub.Samples.get_topic("YOUR_PROJECT_ID", "test-topic")
+
+  """
+  def get_topic(project_id, topic_name) do
+    # Authenticate
+    {:ok, token} = Goth.Token.for_scope("https://www.googleapis.com/auth/cloud-platform")
+    conn = GoogleApi.PubSub.V1.Connection.new(token.token)
+
+    # Make the API request.
+    GoogleApi.PubSub.V1.Api.Projects.pubsub_projects_topics_get(
+      conn,
+      project_id,
+      topic_name
+    )
+    |> case do
```

Comment:
Is this more or less idiomatic?

Reply:
I think it was better before. It's generally considered bad form to start a pipeline with a function call rather than a simple value.

Reply:
Ok, reverted